### PR TITLE
chore: remove extra ts-node dependency

### PR DESCRIPTION
**Pull request by cball**
It is already included via ts-node-dev.

## Changes

- removes ts-node from package.json template.

## Screenshots

n/a

## Checklist

- [ ] Requires dependency update?
- [x] Generating a new app works

### CODE REVIEW

**Review by cmejet — APPROVED** [review]
Do we need to be worried about the version of ts-node-dev being `pre`?

File: template/package.json.ejs [hunk 1]
```diff
@@ -104,7 +104,6 @@
     "start-server-and-test": "^1.11.2",
     "supertest": "^4.0.2",
     "ts-jest": "^26.1.3",
-    "ts-node": "^8.10.2",
     "ts-node-dev": "^1.0.0-pre.63",
```
Comment (cball) [inline template/package.json.ejs:107]:
```suggestion
    "ts-node-dev": "^1.0.0",
```

**Comment by cball** [comment]
@cmejet not anymore, apparently they released 1.0 today!

**Pull request merged**